_data/roadmap.yml

```diff
@@ -1,60 +1,60 @@
 - year: 2014
   accomplishments:
     - name: Launched on Bitcointalk
-      date: April 18, 2014
+      date: 18 April 2014
```

Comment:
We should probably switch to a more international standard, even if it means updating the other pages. Most of the world uses DMY https://en.wikipedia.org/wiki/Date_format_by_country